contracts/testContracts/TestFlowIDTableStaking.cdc

```diff
@@ -0,0 +1,188 @@
+/*
+
+    TestFlowIDTableStaking
```

Comment:
There are a bunch of other setup tasks, ending staking, unstaking, and paying rewards that need to be done with the real staking contract before accounts can call all of the functions and I didn't want to spend the time adding all the setup tests for that since it has already been tested, so I just made the test one to have all the same function signatures and returns so I can just test the interface. Is that ok?